README.md

```diff
@@ -1,163 +1,178 @@
 # Four Keys
 
-
 # Background
 
-Through six years of research, the [DevOps Research and Assessment (DORA)](https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling) team has identified four key metrics that indicate the performance of a software development team.  This project allows you to collect your data and compiles it into a dashboard displaying these key metrics. 
-
+Through six years of research, the [DevOps Research and Assessment (DORA)](https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling) team has identified four key metrics that indicate the performance of a software development team. Four Keys allows you to collect data from your GitHub or GitLab project and compiles it into a dashboard displaying these key metrics.
 
+These four key metrics are:
 
 *   **Deployment Frequency**
 *   **Lead Time for Changes**
 *   **Time to Restore Services**
 *   **Change Failure Rate**
 
+# Who should use Four Keys
 
-# How it works
-
+Use Four Keys if:
 
+*   You want to measure your team's software delivery performance. For example, you may want to track the impact of new tooling or more automated test coverage, or you may want a baseline of your team's performance. 
+*   You have a project in GitHub or GitLab.
+*   Your project has deployments.
 
-1.  Events are sent to a webhook target hosted on Cloud Run
-1.  The Cloud Run target publishes all events to Pub/Sub
-1.  A Cloud Run instance is subscribed to the topics and does some light data transformation and inputs the data into BigQuery
-1.  Nightly scripts are scheduled in BigQuery to complete the data transformations and feed into the dashboard
+Four Keys does not work well with projects that have software releases because of how GitHub and GitLab collects data about releases. 
```

Comment:
So it's best to avoid "currently" in tech docs, because you're implying some future change without being explicit about it. I removed "currently" from a couple other places in the readme. https://developers.google.com/style/future

I'm okay to be flexible on this, especially because it's an open source project. It may be helpful to have this problem / possible solutions tracked in an issue, and we can link to that from the readme (for people who want more info, and who knows maybe someone will want to help address it?).

Do you think we should define deployment vs release here, or will most people get it? I'm sure there are details about how the data is handled that'll be good to get into in the issue tracking this problem.

+1 to roadmap doc, and would be good to link to the roadmap from the contribution info.

Reply:
I think it's better to say "Four Keys works well with projects that have deployments.  Projects with releases and no deployments - eg libraries - do not work well because of how GitHub and GitLab present their data about releases."